typescript/api-client/projects/vcd/schematics/src/schematics/plugin-seed/templates/9.1/__name@dasherize__/src/main/simple-plugin.module.ts

```diff
@@ -27,7 +27,7 @@ const ROUTES: Routes = [
     exports: [],
     providers: [VcdApiClient]
 })
-export class SimplePluginModule extends PluginModule {
+export class <%= classify(module) %>Module extends PluginModule {
```

Comment:
doesn't the name of this file also need to change? it's still hardcoded to simple-plugin.module.ts